schema/pgsql/schema.sql

```diff
@@ -322,9 +320,9 @@ COMMENT ON INDEX idx_incident_history_time_type IS 'Incident History ordered by
 
 CREATE TABLE browser_session (
     php_session_id varchar(256) NOT NULL,
-    username citext NOT NULL,
-    user_agent text NOT NULL,
-    authenticated_at bigint NOT NULL DEFAULT (EXTRACT(EPOCH FROM CURRENT_TIMESTAMP) * 1000),
+    username varchar(254) NOT NULL,
+    user_agent varchar(255) NOT NULL,
+    authenticated_at bigint NOT NULL,
```

Comment:
@ncosta-ic This breaks the Web part btw.

Reply:
@Al2Klimov I do not quite get why you would even change those fields as they are solely used by the PHP Daemon and not touched by the Go Daemon at all.
I know that you're trying to port it over to MariaDB but you need to find appropriate equivalents. I opted for using `citext` for the username field as it was mentioned by Julian in https://github.com/Icinga/icinga-notifications/pull/136#discussion_r1409326979.

As for the `user_agent` field, you would need to change it to a `varchar(4096)` and not only to a `varchar(255)`.
I am personally still more convinced that a device hash would be more appropriate, but I don't make the rules.

Reply:
I meant Web will have to supply authenticated_at by itself.

Reply:
Right, that's why there's the default value. And it would still break when a browser supplies a user_agent  longer than 255 characters as the php daemon allows up to 4096 chars.

Reply:
> As for the `user_agent` field, you would need to change it to a `varchar(4096)` and not only to a `varchar(255)`.

I'm sorry, Noe. I'm afraid I can't do that:

`ERROR 1071 (42000): Specified key was too long; max key length is 3072 bytes`

Reply:
> > As for the `user_agent` field, you would need to change it to a `varchar(4096)` and not only to a `varchar(255)`.
> 
> I'm sorry, Noe. I'm afraid I can't do that:
> 
> `ERROR 1071 (42000): Specified key was too long; max key length is 3072 bytes`

And that's one of the reasons I initially went for a `jooat` hash, but that approach was turned down by others :)